libs/database/src/lib/entities/localization-keys.entity.ts

```diff
@@ -0,0 +1,35 @@
+import {
+  AllowNull,
+  AutoIncrement,
+  Column,
+  Model,
+  PrimaryKey,
+  Table,
+  Unique
+} from "sequelize-typescript";
+import { BIGINT, NUMBER, STRING } from "sequelize";
+
+
+
+@Table({ tableName: "localization_keys", underscored: true, paranoid: false })
+export class LocalizationKeys extends Model<LocalizationKeys> {
+
+  @PrimaryKey
+  @AutoIncrement
+  @Column(BIGINT.UNSIGNED)
+  override id: number;
+
+  @AllowNull
+  @Column(STRING)
+  key: string | null;
+
+  @AllowNull
+  @Column(STRING)
+  value: string | null;
+
+  @AllowNull
+  @Unique
```

Comment:
I'm not sure if these value ids are required to be unique or not, but I do see that the current DB schema doesn't set a uniqueness constraint on this column, so we shouldn't here either.